corehq/apps/app_manager/view_helpers.py

```diff
@@ -24,8 +26,10 @@ def app(self):
             # if get_app is mainly used for views,
             # maybe it should be a classmethod of this mixin? todo
             return get_app(self.domain, self.app_id)
-        except Exception:
-            pass
+        except Http404 as e:
+            raise e
+        except Exception as e:
+            soft_assert(notify_admins=True)
```

Comment:
i don't think this works, does it? `soft_assert` returns a callable which you have to call in order to actually do anything. I think it should be `soft_assert(notify_admins=True)('Unexpected error accessing app!')`